tval

```diff
@@ -48,6 +48,10 @@ var metadata = {
 
 // The module will evaluate and have process.argv ['node', filename, ...]
 System.import(moduleName, {metadata: metadata}).catch(function(err) {
-  console.error('tval fails:',  err.stack || err + '');
+  if (err instanceof Array) {
+    console.log(err.join('\n'));
```

Comment:
With the current output I think that might be bad because it would add a lot of global bindings... One per module.

Maybe we should make `--modules=inline` produce a single IIFE?